### PR TITLE
Update Ipsos test with consent

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?

This adds the Ipsos consent check to the test article.

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [x] Yes (please indicate your plans for DCR Implementation)

### CODE REVIEW

**Comment by shtukas** [comment]
Looks good +1

File: static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js [hunk 1]
```diff
@@ -21,25 +21,17 @@ const IpsosTagging = function () {
...
-        }
-    });
-
-     */
+        onConsentChange(state => {
+            console.log(getConsentFor('ipsos', state));
+            // Initial testing only
+            if(document.location.href === "https://www.theguardian.com/science/grrlscientist/2012/aug/07/3") {
+                if (getConsentFor('ipsos', state)) {
+                    IpsosTagging();
```
Comment (jfsoul) [inline static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js:31]: I think the convention would be to lowercase the first character of the function name. Maybe it could be called `loadIpsosScript` as that is what it does.

File: static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // @flow
-// import { onConsentChange, getConsentFor } from '@guardian/consent-management-platform';
+import { onConsentChange, getConsentFor } from '@guardian/consent-management-platform';
 import config from 'lib/config';
 import { loadScript } from '@guardian/libs';
```
Comment (jfsoul) [inline static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js:5]: Not part of this PR, but if the definitions on line 6 and 7 are just to make linting happy, I'd personally suppress the errors rather than define them. It's a bit unclear otherwise why we have them.

**Review by jfsoul — APPROVED** [review]
